src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2233,6 +2229,10 @@ Uninstall the following packages: {0}?</value>
   <data name="DynamoViewSettingsMenuShowCodeBlockNodeLineNumber" xml:space="preserve">
     <value>Show CodeBlockNode Line Numbers</value>
   </data>
+  <data name="MessageFailedToDownloadPackageVersion" xml:space="preserve">
+    <value>Failed to download version {0} of package with id: {1}.  Please try again and report the package if you continue to have problems.</value>
+    <comment>Message box content. {0} = 1.2.3, {1} = 57d576e8f615e7725800001d</comment>
```

Comment:
are these default values?

Reply:
No, just examples of what kind of text is expected in the placeholders. I thought that's a good way of communicating that to whoever does the text translations.